pandas/computation/pytables.py

```diff
@@ -188,10 +188,6 @@ def stringify(value):
             if v.tz is not None:
                 v = v.tz_convert('UTC')
             return TermValue(v, v.value, kind)
-        elif (isinstance(v, datetime) or hasattr(v, 'timetuple') or
```

Comment:
@jreback - any idea what this was supposed to be doing?  This doesn't impact any tested behavior and not sure it ever would have worked.